diff --git a/src/condor_starter.V6.1/vanilla_proc.cpp b/src/condor_starter.V6.1/vanilla_proc.cpp
index e0895366c9..dfd9e4853b 100644
--- a/src/condor_starter.V6.1/vanilla_proc.cpp
+++ b/src/condor_starter.V6.1/vanilla_proc.cpp
@@ -1405,6 +1405,7 @@ int VanillaProc::streamingOpenFlags( bool isOutput ) {
 	}
 }
 
+#if defined(HAVE_EXT_LIBCGROUP)
 #ifdef LINUX
 void 
 VanillaProc::setCgroupMemoryLimits(const char *cgroup) {
@@ -1515,3 +1516,4 @@ VanillaProc::setCgroupMemoryLimits(const char *cgroup) {
 		}
 }
 #endif
+#endif
diff --git a/src/condor_starter.V6.1/vanilla_proc.h b/src/condor_starter.V6.1/vanilla_proc.h
index 28a8122f36..914c92b859 100644
--- a/src/condor_starter.V6.1/vanilla_proc.h
+++ b/src/condor_starter.V6.1/vanilla_proc.h
@@ -99,8 +99,10 @@ public:
 			@return true if success, false if failure
 		*/
 	virtual bool PublishUpdateAd( ClassAd* ad );
+#if defined(HAVE_EXT_LIBCGROUP)
 #ifdef LINUX
 	void setCgroupMemoryLimits(const char *cgroup);
+#endif
 #endif
 
 	virtual std::string CgroupSuffix() { return "";}
