diff --git a/build/cmake/CondorConfigure.cmake b/build/cmake/CondorConfigure.cmake
index 6141d382ea..00a9bec1e7 100644
--- a/build/cmake/CondorConfigure.cmake
+++ b/build/cmake/CondorConfigure.cmake
@@ -917,7 +917,7 @@ endif()

 #####################################
 # Do we want to link in libssl and kerberos or dlopen() them at runtime?
-if (LINUX AND NOT WANT_PYTHON_WHEELS)
+if (NOT DEFINED DLOPEN_SECURITY_LIBS AND LINUX AND NOT WANT_PYTHON_WHEELS)
        set( DLOPEN_SECURITY_LIBS TRUE )
 endif()

@@ -1060,7 +1060,7 @@ endif()

 #####################################
 # Do we want to link in the GSI libraries or dlopen() them at runtime?
-if (HAVE_EXT_GLOBUS AND LINUX AND NOT WANT_PYTHON_WHEELS)
+if (NOT DEFINED DLOPEN_GSI_LIBS AND HAVE_EXT_GLOBUS AND LINUX AND NOT WANT_PYTHON_WHEELS)
        set( DLOPEN_GSI_LIBS TRUE )
 endif()
