diff --git a/docs/conf.py b/docs/conf.py
index a989a2868d..0ac1141b10 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -122,7 +122,7 @@ man_pages = [
     ('man-pages/bosco_stop', 'bosco_stop', u'HTCondor Manual', [u'HTCondor Team'], 1),
     ('man-pages/bosco_uninstall', 'bosco_uninstall', u'HTCondor Manual', [u'HTCondor Team'], 1),
     ('man-pages/classad_eval', 'classad_eval', u'HTCondor Manual', [u'HTCondor Team'], 1),
-    ('man-pages/classads', 'classads', u'HTCondor Manual', [u'HTCondor Team'], 1),
+    ('man-pages/classads', 'classads', u'HTCondor Manual', [u'HTCondor Team'], 7),
     ('man-pages/condor_adstash', 'condor_adstash', u'HTCondor Manual', [u'HTCondor Team'], 1),
     ('man-pages/condor_advertise', 'condor_advertise', u'HTCondor Manual', [u'HTCondor Team'], 1),
     ('man-pages/condor_annex', 'condor_annex', u'HTCondor Manual', [u'HTCondor Team'], 1),
